manifests/charts/gateways/istio-ingress/templates/deployment.yaml

```diff
@@ -223,6 +223,14 @@ spec:
           - name: ISTIO_META_NETWORK
             value: {{ .Values.global.network }}
           {{- end }}
+          {{- if $network_set }}
+          - name: ISTIO_META_NETWORK
+            value: {{ $network_set }}
+          {{- end }}
+          {{- if and (not $network_set) .Values.global.network }}
```

Comment:
Isn't this the same condition as above? Should we just merge it?

Reply:
Sort of - one is if they set it in values.global, one is if they set it themselves. Can be merged to just resolve the final value of the var and then use it.

Reply:
Probably removing this from ingress though.